server/models/module_spec_schema.yaml

```diff
@@ -226,99 +226,121 @@ definitions:
-                <<: *common_ftype_properties
+                <<: *common_param_type_properties
                 type: { const: multicolumn }
-                name: { $ref: '#/definitions/ftypes/_lib/name' }
-                placeholder: { $ref: '#/definitions/ftypes/_lib/placeholder' }
+                name: { $ref: '#/definitions/param_types/_lib/name' }
+                placeholder: { $ref: '#/definitions/param_types/_lib/placeholder' }
                 tab_parameter: { type: string }  # optional
         tab:
-            <<: *common_ftype
+            <<: *common_param_type
             properties:
-                <<: *common_ftype_properties
+                <<: *common_param_type_properties
                 type: { const: tab }
-                name: { $ref: '#/definitions/ftypes/_lib/name' }
-                placeholder: { $ref: '#/definitions/ftypes/_lib/placeholder' }
+                name: { $ref: '#/definitions/param_types/_lib/name' }
+                placeholder: { $ref: '#/definitions/param_types/_lib/placeholder' }
         multitab:
-            <<: *common_ftype
+            <<: *common_param_type
             properties:
-                <<: *common_ftype_properties
+                <<: *common_param_type_properties
                 type: { const: multitab }
-                name: { $ref: '#/definitions/ftypes/_lib/name' }
-                placeholder: { $ref: '#/definitions/ftypes/_lib/placeholder' }
+                name: { $ref: '#/definitions/param_types/_lib/name' }
+                placeholder: { $ref: '#/definitions/param_types/_lib/placeholder' }
         multichartseries:
-            <<: *common_ftype
+            <<: *common_param_type
             properties:
-                <<: *common_ftype_properties
+                <<: *common_param_type_properties
                 type: { const: multichartseries }
-                name: { $ref: '#/definitions/ftypes/_lib/name' }
+                name: { $ref: '#/definitions/param_types/_lib/name' }
         secret:
-            <<: *common_ftype
+            <<: *common_param_type
             required: [ id_name, type, name ]
             properties:
-                <<: *common_ftype_properties
+                <<: *common_param_type_properties
                 type: { const: secret }
                 name:
                     type: string
                     description: "'Connect' button text"
         custom:
-            <<: *common_ftype
+            <<: *common_param_type
             properties:
-                <<: *common_ftype_properties
+                <<: *common_param_type_properties
                 type: { const: custom }
-                name: { $ref: '#/definitions/ftypes/_lib/name' }
-                placeholder: { $ref: '#/definitions/ftypes/_lib/placeholder' }
+                name: { $ref: '#/definitions/param_types/_lib/name' }
+                placeholder: { $ref: '#/definitions/param_types/_lib/placeholder' }
+
+        anyExceptList:
```

Comment:
`anyExceptList` irks me. While renaming `param_types`, can we ponder this for a second? `anyExceptList` describes the _value_, but not its _purpose_ -- and both will change.

Soon enough, we're going to have _groups_ of parameters. Lists can contain groups but not lists; Groups can't contain groups or lists. At that point, `anyExceptList` will be ambiguous. 

The name is for things that go in lists, right? How about `anyInList`? `anyListChild`? (I love `anyListChild`. Goes nicely with `anyGroupChild`.)